style.css

```diff
@@ -0,0 +1,178 @@
+* {
+  padding: 0;
+  margin: 2px;
+  text-decoration: none;
+  list-style: none;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: sans-serif;
+  flex-direction: column;
+  justify-content: flex-start;
+  height: 100vh;
+}
+
+header {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+.navbar {
+  justify-content: space-between;
+}
+
+nav {
+  background-color: white;
+  height: 80px;
+}
+
+nav h1.logo {
+  display: flex;
+  font-size: 18px;
+  line-height: 80px;
+  font-weight: bold;
+  color: #6070ff;
+  text-align: center;
+  align-items: center;
+  justify-content: flex-start;
+  margin: 2%;
+}
+
+nav ul {
+  float: right;
+  margin-right: 20px;
+}
+
+nav ul li {
+  display: inline-block;
+  line-height: 80px;
+  margin: 0 5px;
+}
+
+a:hover {
+  background: #acb7c3;
+  transition: 0.4s;
+}
+
+.nav-items {
+  margin: 16px 0;
+  transition: 0.7s ease;
+}
+
+.intro {
+  position: static;
+  display: flex;
+  align-items: center;
+  align-content: center;
+  flex-direction: column;
+  height: 50vh;
+  justify-content: flex-start;
+  top: 30%;
+  left: 20%;
+  width: 62%;
+}
+
+.homepage {
+  display: flex;
+  justify-content: flex-start;
+  margin: 5% 10%;
+}
+
+section h1 {
+  display: flex;
+  position: static;
+  font-family: sans-serif;
```

Comment:
- [ ] Please use the correct font as described in your chosen template. I think the font is called font `Inter`. Please don't forget to use a generic font like `sans-serif` or `serif` alongside the external fonts. If you don't, you might get some styling errors relating to the generic font family.